system_metrics_collector/src/topic_statistics_collector/subscriber_topic_statistics.hpp

```diff
@@ -301,17 +307,26 @@ class SubscriberTopicStatisticsNode : public system_metrics_collector::MetricsMe
   /**
    * Subscriber to listen to incoming messages on a topic
    */
-  typename rclcpp::Subscription<T>::SharedPtr subscription_;
+  std::vector<typename rclcpp::Subscription<T>::SharedPtr> subscriptions_;
 
   /**
    * Topic name to compute statistics for
    */
-  std::string collect_topic_name_;
+  std::vector<std::string> collect_topic_names_;
 
   /**
    * Topic name to publish collected statistics to
    */
   std::string publish_topic_name_;
+
+public:
+  /**
+   * Returns multiple topic names
+   */
+  std::vector<std::string> getCollectTopicName()
```

Comment:
```suggestion
  std::vector<std::string> GetCollectTopicNames()
```

Format / rename for consistency.